packages/block-library/src/paragraph/style.scss

```diff
@@ -29,7 +29,7 @@
 }
 
 p.has-background {
-	padding: $block-bg-padding--v $block-bg-padding--h;
+	padding: 1.25em 2.375em;
 }
```

Comment:
This is the only change I'm not sure of. If we want to change this, shoudn't we change it at the variable itself? Should we even change it in the first place?

Reply:
Yeah, this part was also troubling me a bit... so I pushed it as a point for discussion.
If users add a small-text, should it have smaller padding than a large-text? If yes, then `em` values make sense. If on the other hand we want consistent paddings that don't depend on the font-size, then `em` doesn't make sense and could be changed to `rem` values.
The reason I didn't change the variables themselves is because they are used in other blocks as well (columns & heading) and the change would affect more styles than intended. :thinking: 

If the padding change is a blocker for this PR I can just remove it and bring it up in a meeting for further discussion.